modules/nw-ovn-k-adminnetwork-policy.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn-k-network-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="adminnetworkpolicy_{context}"]
+= AdminNetworkPolicy
+
+An `AdminNetworkPolicy` (ANP) is a cluster-scoped custom resource definition (CRD). As a {product-title} administrator, you can use ANP to secure your network by creating network policies before creating namespaces. Additionally, you can create network policies on a cluster-scoped level that is non-overridable by `NetworkPolicy` objects.
+
+The key difference between `AdminNetworkPolicy` and `NetworkPolicy` objects are that the former is for administrators and is cluster scoped while the latter is for tenant owners and is namespace scoped.
+
+An ANP allows administrators to specify the following:
+
+* A `priority` value that determines the order of its evaluation. The lower the value the higher the precedence.
+
+* A `subject` that consists of a set of pods that is selected by the policy.
+
+* A list of ingress rules to be applied for all ingress traffic towards the `subject`.
+
+* A list of egress rules to be applied for all egress traffic from the `subject`.
+
+[NOTE]
+====
+The `AdminNetworkPolicy` resource is a `TechnologyPreviewNoUpgrade` feature that can be enabled on test clusters that are not in production. The feature is behind the flag `enable-admin-network-policy`. For more information on feature gates and `TechnologyPreviewNoUpgrade` features, see "Enabling features using feature gates" in the "Additional resources" of this section.
```

Comment:
@JoeAldinger From where does this flag **enable-admin-network-policy** come.
oc get featuregate cluster -oyaml
<pre>
.....
spec:
  featureSet: TechPreviewNoUpgrade
status:
  featureGates:
  - disabled:
    - name: ClusterAPIInstall
    - name: DisableKubeletCloudCredentialProviders
    - name: EventedPLEG
    - name: MachineAPIOperatorDisableMachineHealthCheckController
    enabled:
    - name: AdminNetworkPolicy
    - name: AlibabaPlatform
    - name: AutomatedEtcdBackup
    - name: AzureWorkloadIdentity
....
</pre>

Reply:
Isn't it listed right there. Or is that saying is isn't behind a feature gate and is default enabled.

Reply:
name is AdminNetworkPolicy. Not sure I have seen enable-admin-network-policy. Also you enable all techpreview features not specifically one of them.

Reply:
Ok, I see what you are saying now. I can clean this up. Thank you.